docs/docs/guides/troubleshooting.md

```diff
@@ -73,3 +73,37 @@ or
 ```bash
 npm install
 ```
+
+### Automatic workletization of gesture callbacks
+
+Reanimated's Babel plugin is setup in a way that automatically marks callbacks passed to gestures in the configuration chain as worklets. This means that as long as all your callbacks are defined in a single chain, you don't need to add a `'worklet';` directive at the beginning of the functions. Here is an example that will be automatically workletized:
```

Comment:
i think it could be useful to give people links to concepts like:
**workletization**
https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/glossary#to-workletize
**reanimated babel plugin**
https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/glossary#reanimated-babel-plugin

Reply:
Done in [1aec22a](https://github.com/software-mansion/react-native-gesture-handler/pull/3274/commits/1aec22a51502862792584f212840f2d23e8d67c4)